content/management/configuration-api/v3.4/index.mdx

```diff
@@ -2924,22 +2923,23 @@ Informs that a chat was deactivated by closing the currently open thread.
 
 <Text>
 
-### `chat_access_granted`
+### `chat_access_updated`
 
-Informs that new, single access to a chat was granted. The existing access isn't overwritten.
+Informs that a chat's access was changed. It contains full access of the chat.
```

Comment:
Maybe "current" access? "Full" access feels like it might be easily misinterpreted.

Reply:
As for the first part, here's my proposal:

> Informs about the update of a user's access to a particular chat.

It's not really _chat's access_, but a user's access to the chat.

For the second part, maybe:

> The webhook contains the updated [access data structure](/messaging/agent-chat-api/#access-1).